public/app/plugins/panel/geomap/components/Tooltip.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { PureComponent } from 'react';
+import { stylesFactory } from '@grafana/ui';
+import { Map } from 'ol';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from '@emotion/css';
+import { config } from 'app/core/config';
+import tinycolor from 'tinycolor2';
+
+interface Props {
+  map: Map;
+}
+
+interface State {
+  visible: boolean;
+  x?: number;
+  y?: number;
+}
+
+export class Tooltip extends PureComponent<Props, State> {
+  style = getStyles(config.theme);
+
+  constructor(props: Props) {
+    super(props);
+    this.state = { visible: false };
+  }
+
+  updateViewState = (onFeature: boolean) => {
+    this.setState({
+      visible: onFeature,
+    });
+  };
+
+  componentDidMount() {
+    const { map } = this.props;
+
+    map.on('pointermove', (evt: any) => {
+      const pixel = map.getEventPixel(evt.originalEvent);
+      const feature = map.forEachFeatureAtPixel(pixel, function (feature) {
+        return feature;
+      });
```

Comment:
```suggestion
      map.forEachFeatureAtPixel(pixel, (feature) => {
        // do your work in here...
        // this will iterate over every feature that is at a pixel -- there may be multiple if multiple layers exist
      });
```